app/models/library_staff_record.rb

```diff
@@ -19,24 +20,23 @@ class LibraryStaffRecord < ApplicationRecord
   # rubocop:disable Metrics/AbcSize
   # rubocop:disable Metrics/MethodLength
   def self.new_from_csv(row)
+    title = row[13]
```

Comment:
@sandbergja is this intentional?

Reply:
Yes, but I'm definitely open to other options.  I wanted to make it clearer that it is used twice below (for both `library_title` and `title`.

Reply:
ok now I see further down . ignore my question